jenkins/custom-ci/generic/cortx-rgw.groovy

```diff
@@ -52,6 +52,13 @@ pipeline {
                     mv ceph*tar.bz2 /root/rpmbuild/SOURCES/                    
                 popd
                 '''
+                sh label: 'Configure yum repositories', script: '''
+                    set +x
+                    yum-config-manager --add-repo=http://cortx-storage.colo.seagate.com/releases/cortx/github/integration-custom-ci/rockylinux-8.4/custom-build-36/cortx_iso/
```

Comment:
can we remove hardcoded url?

Reply:
RGW changes are part of https://github.com/Seagate/cortx-re/pull/826 . I will fix it there. Currently, custom-ci does not call cortx-rgw build.